tests/django/conftest.py

```diff
@@ -0,0 +1,55 @@
+import django
+import pytest
+from django.conf import settings
+from django.core.management.color import no_style
+from django.core.management.sql import sql_flush
+from django.db import connection
+
+settings.configure(
+    SECRET_KEY="not_very",
+    ROOT_URLCONF="tests.django.proj.urls",
+    INSTALLED_APPS=[
+        "django.contrib.auth",
+        "django.contrib.contenttypes",
+        "django.contrib.sessions",
+    ],
+    MIDDLEWARE=[
+        "django.contrib.sessions.middleware.SessionMiddleware",
+        "django.contrib.auth.middleware.AuthenticationMiddleware",
+    ],
+    DATABASES={
+        "default": {
+            "ENGINE": "django.db.backends.sqlite3",
+            "NAME": ":memory:",
+        }
+    },
+)
+
+django.setup()
```

Comment:
Django needs settings to be configured, this is the simplest way to configure django (we also add the required apps to get auth and sessions working)